README.md

```diff
@@ -77,6 +77,14 @@ On FreeBSD you can install a prebuilt `hub` package with
 # pkg install hub
 ```
 
+#### Debian
+
+On Debian (versions sid and buster) you can install `hub` from official repository:
```

Comment:
it should be "from **the** official…"

Please also correct it above where you copied it from